kie-pmml-bom/pom.xml

```diff
@@ -65,6 +65,11 @@
   <dependencyManagement>
     <dependencies>
       <!-- PMML -->
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-pmml-api</artifactId>
+        <version>${version.org.kie}</version>
```

Comment:
Can you please duplicate this dependency (and all the other of the file) with `<classifier>sources</classifier>`?

Reply:
@danielezonca 
What's the reason for that ? Duplicating all of them would make maintaining harder, I would prefer to avoid if not really needed/requested